Asset-Management-Policy.md

maureenlsourcegraph opened the pull request:
Create Asset Management Policy

maureenlsourcegraph committed:
Asset-Management-Policy.md

maureenlsourcegraph committed:
squash! Prettier

maureenlsourcegraph committed:
Merge branch 'main' into asset-management-policy

maureenlsourcegraph committed:
Merge branch 'main' into asset-management-policy

maureenlsourcegraph committed:
Update index.md

nickyvm reviewed (commented):
Two things:
1. can you please add an effective date and a last updated date
2. Ownership of Assets - I don't really understand this sentence and what it means

maureenlsourcegraph committed:
Update asset-management-policy.md

maureenlsourcegraph commented:
> Two things:
> 
> 1. can you please add an effective date and a last updated date
> 2. Ownership of Assets - I don't really understand this sentence and what it means

Added dates and changed language around  ownership > assignment.

maureenlsourcegraph committed:
Merge branch 'main' into asset-management-policy

maureenlsourcegraph committed:
Merge branch 'main' into asset-management-policy

maureenlsourcegraph committed:
Merge branch 'main' into asset-management-policy
